tests/integrations/test-remote-data-blocks.php

```diff
@@ -215,4 +215,67 @@ public function test_get_versions_ignores_non_matching_directories(): void {
 		$this->assertIsArray( $versions );
 		$this->assertEmpty( $versions );
 	}
+
+	/**
+	 * @dataProvider wp_version_support_provider
+	 */
+	public function test_is_supported_wp_version( string $current_wp_version, ?array $config, bool $expected_result ): void {
+		/** @var MockObject|RemoteDataBlocksIntegration $integration_mock */
+		$integration_mock = $this->getMockBuilder( RemoteDataBlocksIntegration::class )
+			->setConstructorArgs( [ $this->slug ] )
+			->onlyMethods( [ 'get_env_config', 'get_wp_version' ] )
+			->getMock();
+
+		$integration_mock->method( 'get_env_config' )->willReturn( $config ?? [] );
+		$integration_mock->method( 'get_wp_version' )->willReturn( $current_wp_version );
+		$this->assertEquals( $expected_result, $integration_mock->is_supported_wp_version() );
+	}
+
+	public static function wp_version_support_provider(): array {
+		return [
+			'no minimum_wp_version in config' => [ '6.7', [], true ],
+			'wp_version less than minimum'    => [ '5.6', [ 'minimum_wp_version' => '6.7' ], false ],
+			'wp_version equal to minimum'     => [ '6.7.1', [ 'minimum_wp_version' => '6.7' ], true ],
+			'wp_version greater than minimum' => [ '6.8.2', [ 'minimum_wp_version' => '6.7' ], true ],
```

Comment:
Could use some tests with patch versions and equalities
```suggestion
			'wp_version equal to minimum' => [ '6.7', [ 'minimum_wp_version' => '6.7' ], true ],
			'wp_version equal to minimum but more specific' => [ '6.7.0', [ 'minimum_wp_version' => '6.7' ], true ],
			'wp_version equal to minimum but less specific' => [ '6.7', [ 'minimum_wp_version' => '6.7.0' ], true ],
```

Reply:
> 'wp_version equal to minimum but less specific' => [ '6.7', [ 'minimum_wp_version' => '6.7.0' ], true ]

This fails since `version_compare` treats `6.7` as less than `6.7.0`. Is this something which might happen with wordpress versions ? If yes, we could patch the logic to handle such cases. When I check the WP releases page - https://wordpress.org/download/releases/, the versions were like `6.7`, `6.7.1`, `6.7.2` with `6.7.0` not mentioned. That might indicate we don't get cases like `6.7.0` but I don't see anything wrong with that. If we decide to patch it should we only worry about patch version being absent/present or extend that for other cases where versions contains texts like `release`, `rc`, `beta` ?

Reply:
I don't know the answer, kinda why I wanted more test cases. I bet @rinatkhaziev has good advice here

Reply:
WP doesn't follow semver. So it's '6.7' => '6.7.1'. pre-releases are `X.Y-beta1', 'X.Y-rc1`, etc.

Reply:
Added few more tests for pre-release versions, but the way it is now, works for above scenarios